src/argus_htmx/destinations/forms.py

```diff
@@ -71,6 +71,47 @@ def _validate_serializer(self):
         return self.cleaned_data
 
 
+class DestinationFormUpdate(DestinationFormCreate):
+    def __init__(self, *args, **kwargs):
+        if instance := kwargs.get("instance"):
+            settings_key = _get_settings_key_for_media(instance.media)
+            instance.settings = instance.settings.get(settings_key)
+        super().__init__(*args, **kwargs)
```

Comment:
add comment explaining that this is done to convert the settings value from the whole dict to just the email/phone number. That way we dont get the entire dict in the frontend, and just the value we are interested in. This is opposite to what is done in the create form, where we translate from the value itself to a whole dict